eng/pipelines/common/platform-matrix.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       targetRid: linux-musl-arm
       platform: Linux_musl_arm
       container:
-        image: ubuntu-18.04-cross-arm-alpine-20200818211451-14441ae
+        image: ubuntu-16.04-cross-arm-alpine-20210409142327-044d5b9
```

Comment:
Any reason we downgraded to 16.04? This will go out of support this month.

Reply:
Yes, the reason was that cross tools (e.g. crossgen targetting arm, but running on x64) would crash on 16.04 and we also use 16.04 on arm64 musl. I guess that when 16.04 goes out of support, we will need to change all 16.04 based images to 18.04 ones.
Ideally, we would run cross build on CentOS 7, but unfortunately that doesn't work due to some differences in default search paths for libs / headers that are compiled into the llvm tools. I have tried to make it work on CentOS 7 in the past, but I didn't have time to finish it. I'll try to dig out that experimental changes once I get some priority things off my table and see if I can make it work.